### PR TITLE
refs #4841 - change ruby193 SCL paths to tfm SCL

### PR DESCRIPTION
**Pull request by domcleal**
Dependent on https://github.com/theforeman/foreman-packaging/pull/249.

### CODE REVIEW

File: foreman.fc [hunk 1]
```diff
@@ -39,8 +39,8 @@
 
 # Passenger SCL file contexts
 
-/opt/rh/ruby193/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
-/opt/rh/ruby193/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/opt/theforeman/tfm/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/opt/theforeman/tfm/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
```
Comment (lzap) [inline foreman.fc:43]: Don't we use passenger from RHSCL?

Looks good, ping when this needs to be merged.

Comment (domcleal) [inline foreman.fc:43]: No, we've always used our own build of it and will continue to for now at least.  There is a passenger40 collection, but adopting that's a whole other set of changes!

Comment (domcleal) [inline foreman.fc:43]: > Looks good, ping when this needs to be merged.

Thanks, this may be merged now as Foreman core is built under 'tfm'.

**Comment by lzap** [comment]
Merged as 3a6366ab5520afefe796ec9d1c7f99d0b23273d0, thank you!

**Pull request closed without merging**